.coveragerc

```diff
@@ -10,6 +10,7 @@ omit =
     # omit pieces of code that rely on external devices being present
     homeassistant/components/accuweather/__init__.py
     homeassistant/components/accuweather/const.py
+    homeassistant/components/accuweather/sensor.py
```

Comment:
There are unrelated commits in the branch.

Reply:
Reverting the wrong merge commit probably didn't removed those. Any help?

Reply:
I usually to do an interactive rebase on latest dev branch. Then you can pick the commits to keep and remove the rest.